src/sendMessage.js

```diff
@@ -7,10 +7,15 @@ var log = require("npmlog");
 module.exports = function(mergeWithDefaults, api, ctx) {
   return function sendMessage(msg, thread_id, callback) {
     if(!callback) callback = function() {};
-    if(typeof msg !== "string") return callback({error: "Message should be of type string and not " + typeof msg + "."});
+    if(typeof msg !== "string" && typeof msg !== "object")
```

Comment:
Well typeof works for strings, booleans and numbers. It just won't differentiate arrays from objects, and for arrays you have `Array.isArray(arr)`, so all we really need is the check for the object case. Maybe put in a helper function, not sure...